test/test_account_predeployed.py

```diff
@@ -1,14 +1,48 @@
 """Predeployed account tests"""
 
+from test.settings import GATEWAY_URL
+
 import pytest
+import requests
 
 from starkware.starknet.core.os.class_hash import compute_class_hash
 
 from starknet_devnet.account import Account
-from .util import assert_equal
+from .util import assert_equal, devnet_in_background
+from .support.assertions import assert_valid_schema
+
+ACCOUNTS_SEED_DEVNET_ARGS = [
+    "--accounts",
+    "3",
+    "--seed",
+    "123",
+    "--gas-price",
+    "100",
+    "--initial-balance",
+    "1_000",
+]
+
 
 @pytest.mark.account_predeployed
 def test_precomputed_contract_hash():
     """Test if the precomputed hash of the account contract is correct."""
     recalculated_hash = compute_class_hash(contract_class=Account.get_contract_class())
     assert_equal(recalculated_hash, Account.HASH)
+
+
+@pytest.mark.account_predeployed
+@devnet_in_background()
+def test_predeployed_accounts_endpoint():
+    """Test if the endpoint return list of predeployed accounts"""
+    response = requests.get(f"{GATEWAY_URL}/predeployed_accounts")
+    assert response.status_code == 200
+    assert_valid_schema(response.json(), "predeployed_accounts.json")
+
+
+@pytest.mark.account_predeployed
+@devnet_in_background(*ACCOUNTS_SEED_DEVNET_ARGS)
+def test_predeployed_accounts_predefined_values():
```

Comment:
Looking at this now, I'm not sure what is the benefit of first testing the schema and then the content of the response.

Reply:
This test that the preseed account method didn't change, or that the values it generated are the same as expected. There are no huge benefits but one is a test of calculate_contract_address_from_hash function.
This was a dilemma why do any test besides schema validation here.

Reply:
But doesn't this second test cover everything that is tested with the first test (the schema test)?

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/138/commits/2c6f53f053bb33749b29b2948fc180c45488bf7d